common/Datasets/direct-lake-dataset.md

```diff
@@ -15,17 +15,20 @@ applies_to:
     - edition: Enterprise
 ---
 
-# Direct Lake Datasets
-Direct Lake datasets connect directly to data sources stored in [Fabric One Lake](https://learn.microsoft.com/en-us/fabric/onelake/onelake-overview). 
+# Direct Lake Semantic Models
+Direct Lake semantic models connect directly to data sources stored in [Fabric One Lake](https://learn.microsoft.com/en-us/fabric/onelake/onelake-overview). 
 
 > [!IMPORTANT]
 > Changing a Direct Lake dataset through the XMLA endpoint will block your ability to change the Direct Lake dataset inside the Fabric Service. Only XMLA endpoint can then edit the Direct Lake This is one of the current limitations of this preview 
 feature.
 
-Tabular Editor 3 can create and connect to this type of dataset. For a tutorial on this please refer to our blog article: [Direct Lake Datasets: How to use them with Tabular Editor](https://blog.tabulareditor.com/2023/08/23/fabric-direct-lake-dataset)
+Tabular Editor 3 can create and connect to this type of dataset. For a tutorial on this please refer to our blog article: [Direct Lake semantic models: How to use them with Tabular Editor](https://blog.tabulareditor.com/2023/08/23/fabric-direct-lake-dataset)
 
-Tabular Editor 2 can connect to Direct Lake datasets, but does not have any built in functionality to create new tables or direct lake datasets. This needs to be done manually or with a C# script. 
+Tabular Editor 2 can connect to Direct Lake semantic models, but does not have any built in functionality to create new tables or direct lake semantic models. This needs to be done manually or with a C# script. 
 
+> [!NOTE]
+> DirectLake models currently use a collation that is different from regular Power BI import semantic models. This may lead to different results when querying the model, or when referencing object names in DAX code.
+For more information please see this blog post by Kurth Buhler: [Case-sensitive models in Power BI: consequences & considerations](https://data-goblins.com/power-bi/case-specific)
```

Comment:
Add `>` so that the "For more information..." text is kept inside the NOTE infobox

Reply:
It is :)
![image](https://github.com/TabularEditor/TabularEditorDocs/assets/11939114/475dd157-1cc1-4042-8267-27f974b724a2)

Reply:
Hiding in shame 😳